rules/falco_rules.yaml

```diff
@@ -3077,4 +3077,16 @@
 # Application rules have moved to application_rules.yaml. Please look
 # there if you want to enable them by adding to
 # falco_rules.local.yaml.
+- list: docker_binaries
+  items: [dockerd, containerd-shim, "runc:[1:CHILD]"]
+
+- macro: docker_procs
+  condition: proc.name in (docker_binaries)
 
+- rule: Modify Container Entrypoint
+  desc: Detect file write activities on container entrypoint symlink (/proc/self/exe)
+  condition: >
+    open_write and (fd.name=/proc/self/exe or fd.name startswith /proc/self/fd/) and not docker_procs and container
```

Comment:
During the time I tested the rule I also see some noise with proc.cmdline = "runc:[1:CHILD] init". Do you mind add it in the condition?